docs/docs/installation/pypi.mdx

```diff
@@ -94,14 +101,9 @@ These will now be available when pip installing requirements.
 
 ## Python Virtual Environment
 
-We highly recommend installing Superset inside of a virtual environment. Python ships with
-`virtualenv` out of the box. If you're using [pyenv](https://github.com/pyenv/pyenv), you can install [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv). Or you can install it with `pip`:
-
-```bash
-pip install virtualenv
-```
+We highly recommend installing Superset inside of a virtual environment.
 
-You can create and activate a virtual environment using:
+You can create and activate a virtual environment using the following commands. Ensure you are using a compatible version of python. You might have to explicitly use for example `python3.11` instead of `python3`.
```

Comment:
NOTE: we started using and recommending `uv`. 

Stepping back, it feels like this section of the docs is hard to keep up to date and hoping in the future we can say "use docker compose for dev/POCs, and use helm for k8s, if you're going on metal, we recommend you align with our helm/docker set up"

It's kind of crazy that we have to explain to people how to set up a virtualenv in the Superset doc. "Set up a python env, we don't care how you do it!"

Reply:
Did we need to revise anything on this PR, or should we merge as-is?